extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -871,6 +863,61 @@ export const AzureProvisionDialog: React.FC = () => {
                 onBack();
               }}
             />
+            <PrimaryButton
+              disabled={!formData.creationType}
+              style={{ margin: '0 4px' }}
+              text={formatMessage('Next')}
+              onClick={() => {
+                switch (formData.creationType) {
+                  case 'import':
+                    setPage(PageTypes.EditJson);
+                    setTitle(DialogTitle.EDIT);
+                    break;
+                  case 'create':
+                  case 'generate':
+                  default:
+                    setPage(PageTypes.ConfigProvision);
+                    setTitle(DialogTitle.CONFIG_RESOURCES);
+                    break;
+                }
+              }}
+            />
+          </div>
+        </div>
+      );
+    } else if (page === PageTypes.ConfigProvision) {
+      return (
+        <div style={{ display: 'flex', flexFlow: 'row nowrap', justifyContent: 'space-between' }}>
+          {currentUser ? (
+            <Persona
+              secondaryText={formatMessage('Sign out')}
+              size={PersonaSize.size40}
+              text={currentUser.name}
+              onRenderSecondaryText={onRenderSecondaryText}
+            />
+          ) : (
+            <div
+              style={{ color: 'blue', cursor: 'pointer' }}
+              onClick={() => {
+                clearAll();
+                closeDialog();
+                logOut();
```

Comment:
should it first logout then closeDialog? since closing the dialog will remove this component from react tree (unmount)

Reply:
Unsure. The existing code logs out after closeDialog.